includes/class-donations.php

```diff
@@ -873,6 +888,24 @@ public static function can_use_streamlined_donate_block() {
 		return false;
 	}
 
+	/**
+	 * Whether the WC cart contains a donation product.
+	 *
+	 * @return bool
+	 */
+	public static function is_donation_cart() {
+		if ( ! self::is_platform_wc() ) {
+			return false;
+		}
+		$donation_products_ids = array_values( self::get_donation_product_child_products_ids() );
+		foreach ( WC()->cart->cart_contents as $prod_in_cart ) {
```

Comment:
Getting two PHP warnings on this line when visiting the Reader Revenue wizard:

```
PHP Notice:  Trying to get property 'cart_contents' of non-object in /newspack/app/public/wp-content/plugins/newspack-plugin/includes/class-donations.php on line 901
PHP Warning:  Invalid argument supplied for foreach() in /newspack/app/public/wp-content/plugins/newspack-plugin/includes/class-donations.php on line 901
```

Reply:
Thank you for catching that! fd1b2acf5c35642eabc2ce494499a0c577e205c2 ensures there are product and a cart before iterating.

Reply:
No longer seeing the warnings, thanks!